### DISCUSSION TITLE
Case to Case rollup not working?

### DISCUSSION THREAD

**Discussion by MAEDSFDC** [SFDO-Community/declarative-lookup-rollup-summaries#1347] [Q&A]
Hi All,

We use DLRS for all our rollups. I am trying to do a basic Case to Case rollup, let's say a count of Child Cases but it doesn't work.

I quickly switch to Task to Case and it works first time.

I used the wizard, Case Parent, Case Child, ParentId auto populates as the relationship field. Any ideas? Is it a bug that needs fixing?

Thanks

Matt

**Comment by aheber** [comment]
@MAEDSFDC this project recommends requesting support through the Trailblazer Community group, https://success.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F9300000009O5p. We would ask you to take your help request there first. Very few people monitor these issues and many more of our experts will show up in that community group to help you get to the root cause.

Before closing this issue I'll offer that you'll likely need to include the details of your rollup configuration including any other fields configured and what steps you went through to test the configuration such as Tested Real-time Trigger, Scheduled Item, Full Calculate, etc...
